app/DynamicsAdapter/DynamicsAdapter.Web/SearchAgency/AgencyRequestController.cs

```diff
@@ -135,6 +150,19 @@ public async Task<IActionResult> UpdateSearchRequest(string requestId, [FromBody
                     return StatusCode(StatusCodes.Status500InternalServerError, new { ReasonCode = "error", Message = ex.Message });
                 }
                 _logger.LogInformation("UpdateSearchRequest successfully");
+
+                //try to get EstimatedDate and positionInQueue
+                try
+                {
+                    updatedSearchRequest = await _agencyRequestService.RefreshSearchRequest(updatedSearchRequest.SearchRequestId);
+                }
+                catch (Exception e)
```

Comment:
I would be good for things we call to know the kind of exceptions they throw and handle/log them specifically. Always catching System.Exception says to me these APIs are not well defined.  Will this call throw ArgumentNullException? RefreshSearchRequestException? InvalidOperationExcpetion. Think of built in services and the exceptions they throw.  int.Parse(...) does not throw IOException.  There are well defined exceptions particular APIs throw.